lib/mirage/impl/mirage_impl_block.ml

```diff
@@ -326,10 +326,13 @@ let ccm_block ?nonce_len key =
     | [ block ] ->
         Fmt.str
           {ocaml|let key = %a in
-                 let key = match Astring.String.cut ~sep:"0x" key with
```

Comment:
Can we then remove `astring` from `packages` above?

Reply:
great suggestion, done in 9e3d3cd0